css/selectors/backdrop.json

```diff
@@ -4,6 +4,13 @@
       "backdrop": {
         "__compat": {
           "description": "<code>::backdrop</code>",
+          "matches": {
+            "keywords": [
+              "::backdrop",
+              "::-webkit-backdrop",
+              "::-ms-backdrop"
```

Comment:
Should we just have the unprefixed version? 

I can work with this either way as both options require tools to do some processing, but I feel this duplicates the documentation further down that already notes which prefix different browsers support for the property.

Reply:
Yes, the duplication is a good point. I'm going to drop that. Though this is yet another case where prefix data is a bit troublesome. See #3488 for more of that.